### PR TITLE
[APT-1613] Next steps

### PR DESCRIPTION
**Pull request by ebeneliason**

### CODE REVIEW

**Comment by ebeneliason** [comment]
Here's a direct link to the preview of the new page: https://deploy-preview-94--pensive-meitner-faaeee.netlify.app/docs/intro/next-steps/

File: _docs-sources/intro/next-steps.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+"sidebar_position": 6
+---
+
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
+
+# Next Steps
+
+Now that your foundational knowledge is in place and your workspace is configured, you’re ready to dive in and learn how to deploy production-grade infrastructure. If you’ve purchased a Gruntwork Reference Architecture, use [this guide](/docs/guides/reference-architecture/overview/overview) to get started. Otherwise, view our [courses](/courses) and [guides](/docs/guides/welcome), or check out the [Service Catalog API reference](/docs/reference/services/intro) to learn what's available.
```
Comment (oredavids) [inline _docs-sources/intro/next-steps.mdx:10]: We should covert the links to a relative one.

**Review by oredavids — APPROVED** [review]
LGTM.

File: _docs-sources/intro/next-steps.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <Card
+    title="Deploy a Reference Architecture"
+    href="/docs/guides/reference-architecture/overview/overview"
+  >
+    Learn how to deploy and utilize your Gruntwork Reference Architecture, step
+    by step.
+  </Card>
+  <Card
+    title="Build Your Own Architecture"
+    href="https://github.com/orgs/gruntwork-io/repositories"
```
Comment (eak12913) [inline _docs-sources/intro/next-steps.mdx:22]: What's the ultimate location of this? The intro section of "Build Your Own"? - Should we put a stub in for the intro page here? I guess: How do we not forget to update this?

**Review by eak12913 — APPROVED** [review]
LGTM!

**Pull request merged**